src/LazyLoadingCommand.php

```diff
@@ -50,12 +50,13 @@ public function __construct(string $name, string $commandClass, ContainerInterfa
         $this->setHelp($command->getHelp());
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output) : int
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        return $this->getCommand()->execute($input, $output);
+        // run, vs execute, to ensure parameter and name binding occur correctly
+        return $this->getCommand()->run($input, $output);
```

Comment:
Did we have any issue with execute here? I see that run has more sense, but I am thinking that maybe we should override `run` in the LazyLoadingCommad` - instead of `execute`?

Reply:
The way I implemented the `ParamAwareCommandTrait`, `run()` is when the input gets decorated with a `ParamAwareInputInterface` instance, so I needed to call `run()` from here.

I agree that it makes more sense for the `LazyLoadingCommand` to override `run()` instead of `execute()`. I'll do that, and then leave `execute()` as an empty implementation (as it will now never be called).